csrankings-s.csv

```diff
@@ -580,7 +580,7 @@ Sema Fatma Oktug,Istanbul Technical University,http://web.itu.edu.tr/oktug,3TRcR
 Sema Oktug,Istanbul Technical University,http://web.itu.edu.tr/oktug,3TRcRPkAAAAJ
 Semih Salihoglu,University of Waterloo,https://cs.uwaterloo.ca/~ssalihog,XxQq8nsAAAAJ
 Sen Cheng,Ruhr-University Bochum,https://www.ini.rub.de/the_institute/people/sen-cheng,_dzbEm4AAAAJ
-Sen He,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=SEHE,ySPHD4cAAAAJ
+Sen He,Augusta University,https://www.augusta.edu/ccs/faculty.php,SPHD4cAAAAJ

```

Comment:
Please restore the original home page (https://www.augusta.edu/faculty/directory/view.php?id=SEHE).